pkg/docker/docker.go

```diff
@@ -800,15 +926,18 @@ func (d *stiDocker) RunContainer(opts RunContainerOptions) error {
 			// the container exits normally).  dump port/etc information for the user.
 			dumpContainerInfo(container, d, image)
 		}
+
 		// Return an error if the exit code of the container is
 		// non-zero.
 		glog.V(4).Infof("Waiting for container %q to stop ...", containerName)
-		exitCode, err := d.client.WaitContainer(container.ID)
+		ctx, cancel := getDefaultContext(15 * time.Minute)
```

Comment:
where did this timeout come from and how does it compare with what we were doing previously?